src/app/reducers/index.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  ActionReducer,
+  ActionReducerMap,
+  createFeatureSelector,
+  createSelector,
+  MetaReducer
+} from '@ngrx/store';
+import { environment } from '../../environments/environment';
+
+export interface State {
+
+}
+
+export const reducers: ActionReducerMap<State> = {
+
+};
+
+
+export const metaReducers: MetaReducer<State>[] = !environment.production ? [] : [];
```

Comment:
always try to go from "true" value. in this case check "environment.production ? value1 : value2"

Reply:
replace them later, when implement "login" status in ngrx